src/worker.cpp

```diff
@@ -602,6 +603,10 @@ void Worker::populateInterfaces(const nlohmann::json& interfaceJson,
                             ((*ipzVpdMap).at(record).at(keyword)), encoding);
                         propertyMap.emplace(property, encoded);
                     }
+                    else
+                    {
+                        propertyMap.emplace(property, "");
```

Comment:
Why this is required?

Reply:
Why empty value should be passed? What will happen is nothing is passed?

Reply:
this is the place where handling value of Object type from JSON, ex: extracting common interfaces.
If ipzVpdMap map is not empty, we are getting value from ipzVpdMap.
if ipzVpdMap is empty I wanted to create property for common interfaces while priming inventory with property(like PartNumber, SerialNumber ..) and empty string value.

Reply:
Ok, Can we have a demo of this workflow? Would like to see working of PIM in these scenarios.

Reply:
sure.

Reply:
Removed the above changes, as static properties updated in PIM by default.